### ISSUE TITLE
Improve write example

### ISSUE THREAD

**Issue by taldcroft** [taldcroft/asciitable#25]
User was confused about table write example with dictionaries.  Provide example more like:

```
x = np.array([1, 2, 3])
y = x**2
asciitable.write({'x': x, 'y': y}, 'outfile.dat')
```

**Comment by taldcroft** [comment]
Also make sure to include `names=(...)` in example to specify output order.

**Comment by taldcroft** [comment]
Closed by e8bf868c20acc7db6024d5d56d27b5c91309ce85.
